python/swiftnav/nav_msg.pyx

```diff
@@ -16,8 +16,8 @@ from signal cimport gnss_signal_t
 
 cdef class NavMsg:
 
-  def __cinit__(self):
-    nav_msg_init(&self._thisptr)
+  def __cinit__(self, sid):
```

Comment:
Yeah, so this should be something like this...

```python
def __cinit__(self, GNSSSignal sid):
    nav_msg_init(&self._thisptr, sid._thisptr)
```

since you have to pass the pointer to the method. You should have gotten a typechecking error here on Travis probably.

(Also, thanks for updating the bindings!)